openslides/core/static/js/core/site.js

```diff
@@ -354,6 +360,32 @@ angular.module('OpenSlidesApp.core.site', [
     }
 ])
 
+// Options for TinyMCE editor used in various create and edit views.
+.factory('EditorOptions', [
+    'gettextCatalog',
+    'Mediafile',
```

Comment:
I do not understand what Mediafile has to do with the WYSIWYG-editor for our apps.